tailwind.config.mjs

```diff
@@ -84,20 +86,29 @@ export default {
           },
         },
       },
+      screens: {
+        // Narrow mobile screens
+        'annotator-sm': '240px',
+        // Wider mobile screens/small tablets
+        'annotator-md': '480px',
+        // Tablet and up
+        'annotator-lg': '600px',
```

Comment:
Just re-implementing existing breakpoints in TW here. I'm keeping annotator breakpoints separate for now; we'll see how the various breakpoints on the sidebar side of things compare as we go...